tikv-client/src/main/java/com/pingcap/tikv/region/RegionStoreClient.java

```diff
@@ -1202,6 +1207,16 @@ private void rawDeleteRangeHelper(RawDeleteRangeResponse resp) {
     }
   }
 
+  // checkStartTs will check the start_ts > gc_safe_point to ensure read correctly
+  public void checkStartTs(long startTs) {
+    BackOffer bo = ConcreteBackOffer.newCustomBackOff(BackOffer.PD_INFO_BACKOFF);
+    Long safePoint = pdClient.getGCSafePoint(bo);
+    if (startTs < safePoint) {
+      throw new GrpcException(
```

Comment:
I think "execute time is less than gc life time" is not accurate. I don't have to be.

Reply:
But it is hard for users to know the true safe point. One thing that can ensure is:
1. now- safe point >= gc_life_time
2. transaction execute time = now-start_ts
3. transaction execute time < spark execute time
if  gc_life_time > spark execute time  we can get:
now- safe point >= gc_life_time > spark execute time >transaction execute time=now-start_ts
=> now- safe point > now-start_ts => start_ts > safe point

Reply:
any better error message for this exception?